cooking_recipe.cmake

```diff
@@ -121,8 +121,8 @@ cooking_ingredient (numactl
 
 cooking_ingredient (zlib
   EXTERNAL_PROJECT_ARGS
-    URL https://zlib.net/zlib-1.2.12.tar.gz
```

Comment:
Please move the information in the pull request to the patch changelog.

Also, I see that 1.2.13 is available on zlib.net.

Reply:
yeah, I used it in https://github.com/scylladb/seastar/commit/c9c45086824dd07b9e850b3759ada26f7f057bcc

zlib.net uses the latest version as the unique address but github's ref is always valid.

Reply:
hi @infdahai , could you address Avi's comment of 

> Please move the information in the pull request to the patch changelog.

namely, you could follow the pattern of https://github.com/scylladb/seastar/commit/8783e005c93f9fc6b48e9a12e08cd6fb5d543c82 and revise your commit message, then force-push your change . currently your commit message reads

> update zlib version

which fails to provide the motivation and the context of the change, or how this change addresses the problem(s).